src/App.js

```diff
@@ -1,23 +1,76 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import "./App.css";
+import contactsArr from "./contacts.json";
 
 function App() {
+  const fiveContacts = [...contactsArr].slice(0, 5);
+  const [contacts, setContacts] = useState(fiveContacts);
+
+  const listItems = contacts.map((contact) => {
+    const roundedPopularity = Math.round(contact.popularity * 100) / 100;
+    return (
+      <tr>
+        <td>
+          <img
+            className="contact-img"
+            src={contact.pictureUrl}
+            alt="Contact profile"
+          />
+        </td>
+        <td>{contact.name}</td>
+        <td>{roundedPopularity}</td>
+        {contact.wonOscar ? <td>🏆</td> : <td></td>}
+        {contact.wonEmmy ? <td>🏆</td> : <td></td>}
+      </tr>
+    );
+  });
+
+
+/// NOT READY!
+  function addRandomContact() {
+    let randomNum = Math.floor(Math.random() * contactsArr.length);   
+    setContacts((prevState) => {
+      let returnArr = [];
+      return prevState.push(contactsArr[randomNum]);
+    });
+
```

Comment:
Just one note here, never write functions inside functions like this.

Instead follow this syntax
`const addRandomContact = () => {}`